### PR TITLE
Upgrade SwaggerUI

### PR DESCRIPTION
**Pull request by astagi**
This PR upgrades SwaggerUI to 3.45.1 version

### CODE REVIEW

File: sanic_openapi/ui/index.html [hunk 1]
```diff
@@ -33,14 +33,13 @@
...
-    <script src="./swagger-ui-bundle.js"> </script>
-    <script src="./swagger-ui-standalone-preset.js"> </script>
+    <script src="./swagger-ui-bundle.js" charset="UTF-8"> </script>
+    <script src="./swagger-ui-standalone-preset.js" charset="UTF-8"> </script>
     <script>
     window.onload = function() {
       // Begin Swagger UI call region
       const ui = SwaggerUIBundle({
         url: "./swagger.json",
-        configUrl: './swagger-config',
```
Comment (artcg) [inline sanic_openapi/ui/index.html:43]: Best to keep the config still here

**Comment by artcg** [comment]
Looks good was thinking about doing this also, can also remove the file ui/version

**Comment by astagi** [comment]
> Looks good was thinking about doing this also, can also remove the file ui/version

I removed the `version` file, better to keep a README file with the minimal information about SwaggerUI and its licence, WDYT?

**Comment by artcg** [comment]
Nice, good idea :)

**Review by artcg — APPROVED** [review]

**Pull request merged**